resources/migrations/008-create-tags.edn

```diff
@@ -0,0 +1,13 @@
+{:up ["ALTER TABLE bookmarks DROP COLUMN tags;
+
+      CREATE TABLE tags (
+                          id bigserial primary key,
+                          name text not null unique);
+
+      CREATE TABLE bookmark_tag (
```

Comment:
@samrat I would like you to think through the ramifications of not having a dedicated primary key here. Ideally not having it should suffice for a simple relation table like this, but think through it. E.g. what might change for this to be a problem? And how relevant might that be for you now?

Reply:
@trycatcher There is a primary key on `(tag_id, bookmark_id)` in line 10. I want the combination of (tag, bookmark) to be unique ie. a `bookmark` can only be tagged with a `tag` once.

Reply:
@samrat What i'm after is that you're not using a `surrogate key` here http://www.agiledata.org/essays/keys.html. Should be fine here, but i'm suggesting that you think through it nonetheless